### ISSUE TITLE
Error with update.sh when used by build-client-win script

### ISSUE THREAD

**Issue by Shampra** [SamBouwer/any-docker#19]
Strange error.

I launch build-client-win.sh, which launches the script **_update.sh windows-latest_**. (WSL2 Ubuntu)
But the script gets stuck moving folders protobuf/* to dist/lib, it can't find the directory.
```
renamed 'grpc-server.exe' -> 'dist/anytypeHelper.exe'
mv: target 'dist/lib/' is not a directory
renamed 'json/internalRelations.json' -> 'dist/lib/json/generated/internalRelations.json'
renamed 'json/internalTypes.json' -> 'dist/lib/json/generated/internalTypes.json'
renamed 'json/systemRelations.json' -> 'dist/lib/json/generated/systemRelations.json'
renamed 'json/systemTypes.json' -> 'dist/lib/json/generated/systemTypes.json'
Done
```
I thought there might be a problem with this script, but if I run it manually, it works fine (under Windows/Powershell as well as on WSL2/Ubuntu).

```
renamed 'grpc-server.exe' -> 'dist/anytypeHelper.exe'
renamed 'protobuf/pb' -> 'dist/lib/pb'
renamed 'protobuf/pkg' -> 'dist/lib/pkg'
renamed 'protobuf/protos' -> 'dist/lib/protos'
renamed 'json/internalRelations.json' -> 'dist/lib/json/generated/internalRelations.json'
renamed 'json/internalTypes.json' -> 'dist/lib/json/generated/internalTypes.json'
renamed 'json/systemRelations.json' -> 'dist/lib/json/generated/systemRelations.json'
renamed 'json/systemTypes.json' -> 'dist/lib/json/generated/systemTypes.json'
Done
```

**Comment by SamBouwer** [comment]
@Shampra Could you show from which dir you run `build_heart_client_win.sh`? The make command `make build-client-win` runs the `build_heart_client_win.sh` script and it assumes the `dist` folder is located at ~/any-docker/anytype-clients/anytype-ts/dist. If you run the `build_heart_client_win.sh` manually from any other dir, it should work the same as from the makefile command. The `dist` folder is created as part of pulling the anytype-ts repository, so I can't really think of any reason why the `update.sh` script has any trouble moving files into the dist folder that should be created as part of cloning the anytype-ts repo.

**Comment by SamBouwer** [comment]
@Shampra ok, should be fixed with 83294c29eaeaf734bce78f4a5af3f6dc8eca6913, could you check? I did `rm dist/*` before, and that got rid of the `lib` folder, too. I've changed that to only delete the files in the dist folder as `update.sh` takes care of the rest.

**Comment by Shampra** [comment]
Well done, it works
